app/models/spree/line_item_decorator.rb

```diff
@@ -0,0 +1,6 @@
+Spree::LineItem.class_eval do
```

Comment:
also regarding scoping to line items, if i went and move it to the line item class, i would end up referencing the `Spree::LineItem` everytime i pull in line items for the order no?

I mean the idea is to be able to do `order_instance.line_items_with_tax_rates` i couldn't think of how to make that call like that when i move the method as a scope to `Spree::LineItem`